iGCS/UserInterface/en.lproj/MainStoryboard.storyboard

```diff
@@ -1310,7 +1310,7 @@
         <scene sceneID="cVi-Id-eoD">
             <objects>
                 <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
-                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
```

Comment:
Let's hold off for now. I've got some mission mocks coming your way tonight
and I've not had a chance to play with this.



On Jan 24, 2015, at 5:02 PM, Claudio Natoli <notifications@github.com>
wrote:

In iGCS/UserInterface/en.lproj/MainStoryboard.storyboard
<https://github.com/fightingwalrus/iGCS/pull/159#discussion_r23500895>:

> @@ -1310,7 +1310,7 @@
>          <scene sceneID="cVi-Id-eoD">
>              <objects>
>                  <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
> -                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
> +                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">

@leskowicz <https://github.com/leskowicz> - my thoughts exactly. Going to
let @btgalush <https://github.com/btgalush> and @andymoe
<https://github.com/andymoe> arm-wrestle over the accept/decline button on
this one.

—
Reply to this email directly or view it on GitHub
<https://github.com/fightingwalrus/iGCS/pull/159/files#r23500895>.